ko/lessons/specifics/ets.md

```diff
@@ -0,0 +1,284 @@
+
```

Comment:
@riseshia Oops, It has one line at top of the file. It is not rendered to HTML files and located in the "Korean Lessons" menu.